packages/cli/src/commands/generate/dbAuth/__tests__/dbAuth.test.js

```diff
@@ -1,12 +1,96 @@
 global.__dirname = __dirname
+
+jest.mock('fs')
+
+import fs from 'fs'
 import path from 'path'
 
 // Load mocks
 import '../../../../lib/test'
 
+const realfs = jest.requireActual('fs')
+import Enquirer from 'enquirer'
+
+import { getPaths } from '../../../../lib'
 import * as dbAuth from '../dbAuth'
 
 describe('dbAuth', () => {
+  beforeEach(() => {
```

Comment:
I think it's best to keep it as the `beforeEach` - I did test it with `beforeAll` but it didn't seem to work well. I implemented your suggestion because I agree it's unnecessary to populate the `mockFiles` every test when it doesn't change.

I worry now that I've made the dbAuth test slow - it now highlights the time red on my tests. Hopefully this isn't too bad? I hate to add to an already existing slow tests issue but I just wanted to add sufficient tests to cover my new behaviours.